students/Tianx/Lesson3/List_lab.py

```diff
@@ -0,0 +1,69 @@
+# ------------------------------------------#
+# !/usr/bin/env python3
+# Title: List_Lab.py
+# Desc: Learn the basic ins and outs of Python lists..
+# Tian Xie, 2020-04-05, Created File
+# ------------------------------------------#
+
+# Series 1
+# Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+my_list = ['Apples', 'Pears', 'Oranges', 'Peaches']
+
+# Display the list
+print(my_list)
+
+# Ask the user for another fruit and add it to the end of the list
+response = input("Please enter a name for a fruit:")
+my_list.append(response)
+
+# Display the list
+print(my_list)
+
+# Ask the user for a number and display the number back to the user and the fruit corresponding to that number.
+
+for i,item in enumerate(my_list):
+    i = int(input("Please enter a number:"))-1
+    print(i+1, item)
+    break
+
+# Add another fruit to the beginning of the list using “+” and display the list.
+another_fruit = ['Lemon']
+my_list = another_fruit + my_list
+print(my_list)
+
+# Add another fruit to the beginning of the list using insert() and display the list.
+another_fruit2 ='Strawberry'
+my_list.insert(0,another_fruit2)
+print(my_list)
+
+# Display all the fruits that begin with “P”, using a for loop.
+for item in my_list:
+    if 'P' in item:
+        print(item)
```

Comment:
Couple of things - you may want to use upper() or lower() to catch case sensitive user input.  Also, this is going to catch a 'P' in any part of any word in the list.  Only want fruits that start with 'P'.